src/node.cc

```diff
@@ -1002,7 +1002,9 @@ MaybeLocal<Value> InternalMakeCallback(Environment* env,
   } else {
     std::vector<Local<Value>> args(1 + argc);
     args[0] = callback;
-    std::copy(&argv[0], &argv[argc], &args[1]);
```

Comment:
You could replace `&args[1]` with `&args.data()[1]` to get rid of the conditional.

(It's legal to create a pointer that points one element beyond the array as long as you don't dereference it - which won't happen if `argc == 0` because there won't be elements to copy.)

Reply:
Even better: replace it with `args.begin() + 1`, since `std::copy` works with iterators.

Reply:
Ok, updated now. Thanks for the feedback.